config/prod/prow/jobs/custom/test-infra.yaml

```diff
@@ -81,6 +81,27 @@ presubmits:
           secretName: test-account
 
 periodics:
+# ci-knative-heartbeat job is for monitoring purpose,
```

Comment:
Should be:
`# Used for prometheus, alert(s) will be sent if this job hadn't been succeeded for some time

Reply:
done